### DISCUSSION TITLE
想问下滴滴出行的groupid应该怎么获取

### DISCUSSION THREAD

**Discussion by XuuKoo** [blackmatrix7/ios_rule_script#335] [Q&A]
token、activityid、channelid获取成功，但是groupid没有，执行签到脚本日志中提示：

09:05:06.735] console.log => [WARNING] [滴滴出行]
获取瓜分福利金活动明细果失败，响应异常：{"errno":101,"errmsg":"参数 group_id 缺失","traceid":"手工隐去","data":null}

[09:05:06.735] console.log => [WARNING] [滴滴出行]
获取瓜分福利金act_id失败

**Comment by blackmatrix7** [comment]
滴滴出行的脚本已经暂停维护了。
